### ISSUE TITLE
Would it be possible to get this to run on WiiU RA?

### ISSUE THREAD

**Issue by Ploggy** [zach-morris/same_cdi#4]
I'm very interested to see this in the WiiU RA if possible :) I would have made this issue on the RA repo but there's no tab for issues there so I brought it here.
Would this be able to work on WiiU? I don't know what I would need to edit in the Makefile.libretro file for this to compile on WiiU.

**Comment by zach-morris** [comment]
If MAME can be compiled for the platform, then so can same_cdi.  I dont see MAME in the buildbot, but I'm not sure if thats a limitation because the core is just too huge, or because it wont compile for other reasons.  @jdgleaver may know

**Comment by Ploggy** [comment]
Yes, we don;t have anything from Mame20015 and up, I'm not sure of it was due to some WiiU limitations, maybe..  but if it was, just compiling as same_cdi should be ok right?
